packages/odf/components/odf-dashboard/object-storage-card/capacity-card.tsx

```diff
@@ -8,8 +9,8 @@ import {
   usePrometheusPoll,
 } from '@openshift-console/dynamic-plugin-sdk-internal';
 import { useTranslation } from 'react-i18next';
-import { humanizeBinaryBytes } from '../../../humanize';
-import CapacityCard from '../../common/capacity-card/capacity-card';
+import { humanizeBinaryBytes } from '../../../../shared/utils/humanize';
```

Comment:
Let's leverage the different packages now for cleaner import paths (maybe replacing more than 2 `../` with their package-prefixed paths?).
```suggestion
import { humanizeBinaryBytes } from '@odf/shared/utils/humanize';
```

Reply:
Let's do this only for cross package imports. Same package imports should use relative path.

Reply:
Makes sense.